README.md

```diff
@@ -152,28 +152,36 @@ Now, when we consider time complexity, we never care about how much we need to a
 | ------------- |-------------| ------------- |-------------| ------------- |-------------|
 |n = 1000 |9.9|  1000   |  1,000,000 | 1,000,000,000  |   1,001,001,110 |       
 
-You can see that in our formula of **n^3 + n^2 + n + 100**, when n is 1000, the formula returns 1,001,001,110.
+You can see that in our formula of **n^3 + n^2 + n + log2(n) + 100**, when n is 1000, the formula returns approximately 1,001,001,110. 
 
 Moreover, you can see that compared to n^3, the n^2 doesn't move the dial.  It accounts for just 1,000th of our overall cost.  And this is still when our n is relatively small.  Imagine when our input size increases to ten thousand.  So we see that the leading exponent is dominant when calculating the cost of our function. 
 
-Now if we can exclude something like n^2 when n approaches infinity, we can also exclude anything that we multiply n by.  It just doesn't make the type of impact that we care about.  We care about things that change our formula by a factor of n when **n approaches infinity**.  So compared to that, any number you multiply our formula by will be insignificant.    
+Now if we can exclude something like n^2 when n approaches infinity, we can also exclude anything that we multiply n by. It just doesn't make the type of impact that we care about.  We care about things that change our formula by a factor of n when **n approaches infinity**.  So compared to that, any number you multiply our formula by will be insignificant.    
 
-So in summary, when considering asymptoptic time complexity, we only look to the term with the largest exponent, we only consider the worse case scenario, and we ignore co-efficients as well as any smaller terms.  
+So in summary, when considering asymptoptic time complexity, we only look to the term with the largest exponent, we only consider the worse case scenario, and we ignore coefficients as well as any smaller terms.  
 
 We call this big O.
 
+NOTE: Big O is a little more involved than the intuition presented above. There are two important additional concepts to note. First of all, when people colloquially say big O (pronounced "big oh"), the _technical_ term they usually mean is big ϴ ("big theta"). Big ϴ includes the concept of a "tight asymptotic bound". Using the example above, when our equation was f(n) = n^3 + n^2 + n + log2(n) + 100, big ϴ requires us to describe this with an upper AND lower asymptotic bound. Thus, it would be correct for us to say that ϴ(n) = n^3, but incorrect for us to say ϴ(n) = n^2, and also incorrect for us to say ϴ(n) = n^4. Technically, big O notation would allow us to correctly say that our function f(n) is O(n) = n^3 and O(n) = n^4, and O(n) = n^5...etc. Under big O notation we are only establishing an upper asymptotic bound to our function f(n), and thus it is less precise. For the mathematically-minded of you, the formal mathematical definitions of the two are: 
```

Comment:
```suggestion
Note: Mathematicians and academic computer scientists use different formal definitions for Big O than those presented here. There are two important additional concepts to note. First of all, when people colloquially say "Big O" (pronounced "big oh"), the _technical_ term they usually mean is big ϴ ("big theta"). Big ϴ includes the concept of a "tight asymptotic bound". Using the example above, when our equation was f(n) = n^3 + n^2 + n + log2(n) + 100, ϴ requires us to describe this with an upper AND lower asymptotic bound. Thus, it would be correct for us to say that ϴ(n) = n^3, but incorrect for us to say ϴ(n) = n^2, and also incorrect for us to say ϴ(n) = n^4. Technically, big O notation would allow us to correctly say that our function f(n) is O(n) = n^3 and O(n) = n^4, and O(n) = n^5...etc. Under big O notation we are only establishing an **upper** asymptotic bound to our function f(n), and thus it is less precise. The formal mathematical definitions of the two concepts are: 
```

Reply:
I also think that lots of people who talk about Big O actually do _mean_ Big O, because in practice, tight lower bounds don't matter much to engineers. ϴ matters when you're doing algorithms research, but it's strictly harder to prove.